lxd/storage/backend_lxd.go

```diff
@@ -149,8 +149,8 @@ func (b *lxdBackend) Driver() drivers.Driver {
 
 // MigrationTypes returns the migration transport method preferred when sending a migration,
 // based on the migration method requested by the driver's ability.
-func (b *lxdBackend) MigrationTypes(contentType drivers.ContentType, refresh bool) []migration.Type {
-	return b.driver.MigrationTypes(contentType, refresh)
+func (b *lxdBackend) MigrationTypes(contentType drivers.ContentType, refresh bool, snapshots bool) []migration.Type {
```

Comment:
Please can you update the docblock here explaining what `snapshots` arg indicates and what its used for.